cmd/dep/gopath_scanner.go

```diff
@@ -225,6 +243,7 @@ func (g *gopathScanner) scanGopathForDependencies() (projectData, error) {
 		}
 		v, err := gps.VCSVersion(abs)
 		if err != nil {
+			invalidSVC[pr] = true
 			notondisk[pr] = true
```

Comment:
👍 Yes thanks, we need to set notondisk for missing vcs errors, because we use that later on.